.github/pull_request_template.md

```diff
@@ -1,14 +1,14 @@
-*By submitting this pull request you agree that all contributions to this project are made under the MIT license.*
-
-## Issues
-
-<!-- Link the issues this change resolves, or describe them -->
+<!-- By submitting this pull request you agree that all contributions to this project are made under the MIT license. -->
```

Comment:
Gonna talk to a lawyer about how to properly communicate this. For now, I'd rather make this impossible to miss.

Reply:
Given that Facebook [doesn't even put any disclaimers](https://github.com/reactjs/react.dev/blob/main/.github/PULL_REQUEST_TEMPLATE.md), it's probably not legally required for it to be in the PR template. I will move it to the bottom of the template in small text for now.